File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int pyramidHeight = 0;
    while ((pyramidHeight < 1) || (pyramidHeight > 8)) {
        printf("Input a height for the pyramid (between 1 and 8 inclusive): ");
        scanf("%d", &pyramidHeight);
    }
    for (layer = 1; layer < pyramidHeight + 1; layer++) {
        printf(" "*)
    }
    printf("%i", pyramidHeight);
}

File: pset6/sentimental-mario-more/mario.py
# TODO
max_height = 8
height = 0

while height < 1 or height > max_height:
    height = input("Height: ")
    try:
        height = int(height)
    except:
        height = 0

for layer in range(height):
    for _ in range(height - layer - 1):
        print(" ", end = "")
    for _ in range(layer + 1):
        print("#", end = "")
    print("  ", end = "")
    for _ in range(layer + 1):
        print("#", end = "")
    print()